examples/crm/src/dashboard/TasksList.tsx

```diff
@@ -67,22 +65,92 @@ export const TasksList = () => {
             </Box>
             <Card sx={{ px: 2, mb: '2em' }}>
                 <AddTask selectContact />
-                <ResourceContextProvider value="tasks">
-                    <ListContextProvider value={listContext}>
-                        <TasksIterator showContact />
-                    </ListContextProvider>
-                </ResourceContextProvider>
-                {!isPending && (
-                    <Button
+                <Stack gap={3} mt={2}>
+                    <TaskListFilter
+                        title="Overdue"
+                        filter={taskFilters.overdue}
+                        contacts={contacts}
+                    />
+                    <TaskListFilter
+                        title="Today"
+                        filter={taskFilters.today}
+                        contacts={contacts}
+                    />
+                    <TaskListFilter
+                        title="This week"
+                        filter={taskFilters.thisWeek}
+                        contacts={contacts}
+                    />
+                    <TaskListFilter
+                        title="Later"
+                        filter={taskFilters.later}
+                        contacts={contacts}
+                    />
+                </Stack>
+            </Card>
+        </>
+    );
+};
+
+const TaskListFilter = ({
```

Comment:
**nit-pick:** Maybe move this component to another file as the file starts to get large ?